gui/wxpython/datacatalog/tree.py

```diff
@@ -894,6 +909,13 @@ def InsertMapset(self, name, location_node):
         self._model.SortChildren(location_node)
         self.RefreshNode(location_node, recursive=True)
 
+    def InsertLocation(self, name, grassdb_node):
```

Comment:
This seems currently unused, because ReloadTreeItems adds that node. Just remove it now, it's not correctly implemented anyway, you would need to rebase to master due to #797.

Reply:
Removed.